packages/bundler/src/BundlerCollectorTracer.ts

```diff
@@ -90,22 +94,25 @@ interface BundlerCollectorTracer extends LogTracer, BundlerCollectorReturn {
  */
 export function bundlerCollectorTracer (): BundlerCollectorTracer {
   return {
-    numberLevels: [],
+    topLevelCalls: [],
     currentLevel: null as any,
     keccak: [],
     calls: [],
     logs: [],
     debug: [],
     lastOp: '',
-    numberCounter: 0,
+    // event sent after all validations are done: keccak("BeforeExecution()")
+    stopCollectingTopic: 'bb47ee3e183a558b1a2ff0874b079f3fc5478b7454eacf2bfc5af2ff5878f972',
```

Comment:
If I may suggest, it could be better to calculate this hash here instead of hard-coding it.

Reply:
I would like it too... however, this code is converted into javascript (automatically), so I can't easily even extract it as an external contract
(I could put a placeholder string here, and then replace the placeholder with a calculated value on the javascript text, but I think that's too ugly.)
What I REALLY wanted is have it as a parameter to the script: but tracer doesn't have parameters. it is a single javascript object evaluated by geth parser, and everything has to be members in it.